ehr2vec/feature_importance/utils.py

```
@@ -1,5 +1,7 @@
 import numpy as np
 from typing import Dict, List, Tuple
+import pandas as pd
```

Ruff (0.8.2)

`pandas` imported but unused

Remove unused import: `pandas`

(F401)